src/ctap/data_formats.rs

```diff
@@ -1180,6 +1187,15 @@ mod test {
         );
         let created_cbor: cbor::Value = authenticator_transport.unwrap().into();
         assert_eq!(created_cbor, cbor_authenticator_transport);
+
+        let transports = ["usb", "nfc", "ble", "internal"];
+        for transport in transports.iter() {
```

Comment:
This is not future-proof, but here I really don't see how to do better without procedural macros.

Reply:
Agreed for not being future proof. We can discuss in the comment above about if this is tolerable.